lib/vfscore/include/vfscore/vnode.h

```diff
@@ -144,6 +144,8 @@ struct vattr {
 
 #define IO_APPEND	0x0001
 #define IO_SYNC		0x0002
+#define IO_ASYNC	0x0004
+#define IO_DIRECT	0x0008
```

Comment:
Please add some short comments on these macros, like the one on the `MNT_NOATIME`.